crates/symbolicator-service/src/services/cficaches.rs

```diff
@@ -85,7 +85,6 @@ impl CfiCacheActor {
 
 #[derive(Clone, Debug)]
 struct FetchCfiCacheInternal {
-    request: FetchCfiCache,
```

Comment:
Wait, was the request only passed through for metrics?

Reply:
Yes :-D There is a bunch of things that are only being cloned and passed around for metrics and/or logging.